index.html

```diff
@@ -49,5 +49,169 @@ <h1 class="h1">I'm Ezechukwu moses Glad to see you!</h1>
+
+      <div class="card">
+        <img class="img" src="./image/Snapshoot4.png" alt="ID" />
+        <div class="card-contents"></div>
+        <h2>Mult-Post Stories</h2>
+        <ul class="works">
+          <li>CANOPY</li>
+          <li><img src="./image/Counter.png" alt="Counter" /></li>
+          <li>Back End Dev</li>
+          <li><img src="./image/Counter.png" alt="Counter" /></li>
+          <li>2015</li>
+        </ul>
+        <p class="p">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="works2">
+          <li><img src="./image/Tag.png" alt="html" /></li>
+          <li><img src="./image/Tag1.png" alt="css" /></li>
+          <li><img src="./image/Tag2.png" alt="javaScript" /></li>
+        </ul>
+        <div class="div">
+          <button class="btn-big btn">see project</button>
+        </div>
+      </div>
+    </section>
+    
+    <div class="bg-white">
+      <section id="aboutme">
+        <div class="me">
+          <h3 class="aboutme">About Myself</h3>
+          <p class="p">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If you
+            like what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+  
+          <h4 class="h4-connect">LET'S CONNECT</h4>
+  
+          <ul class="flex item-center">
+            <li class="icon-tag">
+              <a href="#"><img src="./image/twitter.png" alt="twitter" /></a>
+            </li>
+            <li class="icon-tag">
+              <a href="#"><img src="./image/Linkedin.png" alt="linkedin" /></a>
+            </li>
+            <li class="icon-tag">
+              <a href="#"><img src="./image/m.png" alt="m" /></a>
+            </li>
+            <li class="icon-tag">
+              <a href="#"><img src="./image/git.png" alt="Github" /></a>
+            </li>
+            <li class="icon-tag">
+              <a href="#"><img src="./image/angelist.png" alt="angelist" /></a>
+            </li>
+          </ul>
+          <div class="div">
+            <button class="btn-big btn">Get my resume</button>
+          </div>
+        </div>
+        <div class="base-about-myself">
+          <h4>Languages</h4>
+          <ul>
+            <li class="flex item-center bg-languages"><a><img src="image/ellipsejs.png" alt="frameJs" /></a>JavaScript</li>
+            <li class="flex item-center bg-languages"><a><img src="image/ellipsehtml.png" alt="frameJs" /></a>HTML</li>
+            <li class="flex item-center bg-languages"><a><img src="image/ellipsecss.png" alt="frameJs" /></a>CSS</li>
+          </ul>
+          <div>
+            <ul>
+              <li class="frameworks"><p>Frameworks</p> <p>></p></li> <hr>
+              <li class="skills"><p>Skills</p> <p>></p></li>
+              <hr>
+            </ul>
```

Comment:
- [ ] Like my colleague talked about having a design consistent with the benchmark, please consider applying the changes I recommended above in the `about me` section as they are similar.
- To further clarify this, I have provided the figma design and yours, so that you can fix the styling differences. Please do not forget to pick your values from Figma.
- Your Design
![image](https://user-images.githubusercontent.com/99715640/212444836-3942d20a-b02a-4912-a76d-06142971be63.png)
 
- Figma Design:
![image](https://user-images.githubusercontent.com/99715640/212444896-eb5a3dfa-d03a-4409-9d38-f7c346421ab8.png)

Reply:
Thanks for the review @AbayomiOlaoye